frontend_lib/src/component/ShareDownload/NewShareDownload.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react'
+import { withTranslation } from 'react-i18next'
+import Radium from 'radium'
+import { Popover, PopoverBody } from 'reactstrap'
+import { generateRandomPassword } from '../../helper.js'
+
+const color = require('color')
+
+class NewShareDownload extends React.Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      passwordActive: false,
+      popoverOpen: false,
+      hidePassword: true
+    }
+  }
+
+  popoverToggle = () => {
+    this.setState({
+      popoverOpen: !this.state.popoverOpen
+    })
+  }
+
+  handleSeePassword = () => {
+    this.setState({hidePassword: !this.state.hidePassword})
+  }
+
+  handleRandomPassword = () => {
+    this.props.onChangePassword({target: {value: generateRandomPassword()}})
+    const passwordInput = document.getElementsByClassName('shareDownload__password__input')[0]
+    if (passwordInput.type === 'password') {
+      this.handleSeePassword()
+    }
+  }
+
+  handleSeePasswordInput = () => {
+    this.setState({passwordActive: true})
+  }
+
+  render () {
+    const { props, state } = this
+
+    return (
+      <div className='shareDownload__new'>
+        <div className='shareDownload__title'>
+          {props.t('New share')}
+        </div>
+
+        <div className='shareDownload__email'>
+          <textarea
+            type='text'
```

Comment:
textarea tag doesn't have a type attribute
I guess this one has been forgotten during the transition from an input to a textarea